docs/deployment/heroku.md

```diff
@@ -8,6 +8,27 @@ The only thing we'll need for this guide is a working Phoenix application. For t
 
 Our main goal for this guide is to get a Phoenix application running on Heroku.
 
+### Limitations
+
+Heroku is a great platform overall, but for Elixir apps, it comes with a number of limitations you should be aware of. 
```

Comment:
Thank you @jesseshieh!

Listing those limitations are important but they sound like a blocker for the use of Phoenix on Heroku. We need to make it clear that they come to mind when you need to push the envelope. What about something like:

> Heroku is a great platform and Elixir performs well on it. However, you may run into limitations if you plan to leverage advanced features provided by Elixir and Phoenix, such as:

Then for the closing paragraph:

> If you are just getting started or you don't expect to use the features above, Heroku should be enough for your needs. For instance, if you are migrating an existing application running on Heroku to Phoenix, keeping a similar set of features, Elixir will perform just as well or even better then your current stack.